tox.ini

```diff
@@ -1,41 +1,33 @@
 [tox]
 downloadcache = {toxworkdir}/_download/
-envlist = py26-1.4.X,py26-1.3.X,py26-1.2.X,docs
+envlist =
+    {py26,py27}-{django1.6.X},
+    {py27}-{django1.7.X},
+    docs
 
 [testenv]
-commands = django-admin.py test decisiontree
+basepython =
+    py26: python2.6
+    py27: python2.7
+
+commands =
+    coverage run {envbindir}/django-admin.py test decisiontree
+    flake8
+    coverage report -m
```

Comment:
Flake8 does change between Python 2 and Python 3 with respect to valid syntax.

If we want to reduce the number of times these things are run - maybe Flake8 and coverage should be run only for the most up-to-date combination of Python/Django/RapidSMS that we test?